### PR TITLE
Feat/1077/db fixtures cli

### PR DESCRIPTION
**Pull request by gudgl**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: storage/bounces.go [hunk 1]
```diff
@@ -7,3 +7,8 @@ import (
 func (db *store) CreateBounce(b *entities.Bounce) error {
 	return db.Create(b).Error
 }
+
+// DeleteAllBouncesForUSer deletes all bounces for user
+func (db *store) DeleteAllBouncesForUSer(userID int64) error {
```
Comment (FilipNikolovski) [inline storage/bounces.go:12]:
```suggestion
func (db *store) DeleteAllBouncesForUser(userID int64) error {
```

**Review by FilipNikolovski — CHANGES_REQUESTED** [review]

File: fixtures/cmd/clear.go [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+	"github.com/mailbadger/app/services/templates"
+)
+
+var u *entities.User
+
+// clearCmd represents the clear command
+var clearCmd = &cobra.Command{
+	Use:   "clear",
+	Short: "clear deletes all date for the user",
```
Comment (Dzalevski) [inline fixtures/cmd/clear.go:22]: @gudgl 
```suggestion
	Short: "clear deletes all data for the user",
```

File: fixtures/cmd/root.go [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+	// If a config file is found, read it in.
+	if err := viper.ReadInConfig(); err != nil {
+		fmt.Printf("[ERROR %s] failed to read config file", err.Error())
+		os.Exit(1)
+	}
+}
+
+// makeConfigFromEnv creates configuration string for db connection
+func makeConfigFromEnv(driver string) string {
```
Comment (Dzalevski) [inline fixtures/cmd/root.go:96]: @gudgl We have these helper methods in storage/db.go ?

**Review by Dzalevski — CHANGES_REQUESTED** [review]

**Comment by Dzalevski** [comment]
@gudgl It's good to have unit tests for all new storage methods

File: fixtures/cmd/root.go (hunk 3 above)
Comment (gudgl) [reply fixtures/cmd/root.go:96]: The functions in storage db are fetching env variables from the os and here I couldn't make it work with reading the env variables from the os and here I use viper to load configs but I will take a look and try to change it.

**Review by FilipNikolovski — DISMISSED** [review]

**Pull request merged**